addon/sidebar.css

```diff
@@ -147,3 +147,20 @@ body {
   color: rgb(0, 96, 223);
   text-decoration: underline;
 }
+
+/* Dark theme */
+
+.page.dark-theme {
+  color: #fff;
+  background: #4a4a4f;
+}
+
+.page.dark-theme button:hover,
+.page.dark-theme button:focus,
+.page.dark-theme .graphic__shadow {
+  background: #6d6d6f;
```

Comment:
I like the effect on the shadow, but it creates a strange highlighter effect on the links.